OpenRA.Mods.Common/UtilityCommands/ImportLegacyMapCommand.cs

```diff
@@ -271,7 +280,15 @@ void LoadWaypoints(IniSection waypointSection)
 						new OwnerInit("Neutral")
 					};
 
-					Map.ActorDefinitions.Add(new MiniYamlNode("waypoint" + kv.First, ar.Save()));
+					var waypointName = "waypoint" + kv.First;
+					if (kv.First == 25)
```

Comment:
Source:

* https://github.com/OpenRA/OpenRA/wiki/C&C%20Singleplayer%20Maps
* http://cnc-comm.com/files/RACG110.TXT

No, they aren't. Should be fixed now.